pygerduty/v2.py

```diff
@@ -75,27 +75,34 @@ def create(self, **kwargs):
 
         for kwarg_key, kwarg_value in kwargs.iteritems():
             if kwarg_key.endswith('_id'):
-                new_key = kwarg_key[:-3]
-                new_kwargs[new_key] = {
-                    'id': kwarg_value,
-                    'type': new_key
-                }
+                new_kwargs = id_to_obj(kwarg_key, kwarg_value)
             elif kwarg_key.endswith('_ids'):
-                new_key = kwarg_key[:-3]
-                new_kwargs = []
-                for value in kwarg_value:
-                    new_kwarg = {
-                        'id': value,
-                        'type': new_key
-                    }
-                    new_kwargs.append(new_kwarg)
+                new_kwargs = ids_to_objs(kwarg_key, kwarg_value)
             else:
                 new_kwargs[kwarg_key] = kwarg_value
         data[self.sname] = new_kwargs
 
         response = self.pagerduty.request("POST", path, data=_json_dumper(data), extra_headers=extra_headers)
         return self.container(self, **response.get(self.sname, {}))
 
+    @staticmethod
+    def id_to_obj(key, value):
+        new_key = key[:-3]
```

Comment:
@gmjosack I struggled with figuring out where to put the string slice. Originally I left it inside of the create function but found that hard to test. Putting it in both `id_to_obj` and `ids_to_objs` it caused the key to be sliced twice since `ids_to_objs` depends on `id_to_obj`. It created something like this:

```[{'id': 'PF9KMXH', 'type': 'servi'}, {'id': 'PIJ90N7', 'type': 'servi'}]```

My solution was to only do it in `id_to_obj`, but then I realized we check for '_ids' two times (here and in create function) and this did not see very dry. Is there a better way to do this?